code/modules/projectiles/guns/projectile/scout_sniper.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/weapon/gun/projectile/automatic/w556
```

Comment:
I told you to reparent this from `/obj/item/weapon/gun/projectile/automatic/rifle`. Change this line to the following:
`/obj/item/weapon/gun/projectile/automatic/rifle/w556`